controls/ubuntu-18.04-server-cis-5.2.1.rb

```diff
@@ -30,14 +30,14 @@
   "
   impact 0.5
   tag severity: "medium"
-  tag gtitle: nil
-  tag gid: nil
-  tag rid: nil
-  tag stig_id: nil
-  tag fix_id: nil
-  tag cci: nil
   tag nist: ["AC-3 (3)", "Rev_4"]
   tag cis_level: 1
   tag cis_controls: ["14.6", "Rev_7"]
```

Comment:
move the rev_7 into the tag

Reply:
Is this the `tag cis_cdc_version: 7`? Or is there another tag that should be used?

Reply:
yup